test/integration/standup.test.js

```diff
@@ -0,0 +1,196 @@
+const chai = require("chai");
+const { object } = require("joi");
+const { expect } = chai;
+
+const app = require("../../server");
+const addUser = require("../utils/addUser");
+const userQuery = require("../../models/users");
+const userData = require("../fixtures/user/user")();
+const authService = require("../../services/authService");
+const cleanDb = require("../utils/cleanDb");
+const standupData = require("../fixtures/standup/standup");
+
+const cookieName = config.get("userToken.cookieName");
+const superUser = userData[4];
+
+// eslint-disable-next-line mocha/no-skipped-tests
+describe.skip("Test standup api", function () {
+  let superUserId;
+  let superUserAuthToken;
+  let jwt;
+  let userId = "";
+  beforeEach(async function () {
+    userId = await addUser();
+    jwt = authService.generateAuthToken({ userId });
+    superUserId = await addUser(superUser);
+    superUserAuthToken = authService.generateAuthToken({ userId: superUserId });
+  });
+
+  afterEach(async function () {
+    await cleanDb();
+  });
+
+  describe("Test the mark api", function () {
+    it("Checks superuser can mark a user as monitored", function (done) {
+      chai
+        .request(app)
+        .post(`/standup/${userId}`)
+        .set("Cookie", `${cookieName}=${superUserAuthToken}`)
+        .send({
+          monitor: true,
+        })
+        .end((err, res) => {
+          if (err) {
```

Comment:
make this a one-liner `if(err) return done(err)`;  
also do it wherever needed

Reply:
done